structures/message.js

```diff
@@ -0,0 +1,21 @@
+const s = require('superstruct')
+const Uuid = s.define('Uuid', require('is-uuid').v4)
+const { ZuluDateTimeStruct } = require('./lib');
+
+const Message = s.type({
+    id: s.defaulted(Uuid, require('uuid').v4()),
```

Comment:
```suggestion
    id: s.defaulted(Uuid, uuid.v4),
```

Put require at the top `const uuid = require('uuid');